doc/adr/0002-continuously-delivery-travis-ci.md

```diff
@@ -4,7 +4,7 @@ Date: 2018-10-09
 
 ## Status
 
-Accepted
+[Superseded](0007-github-actions.md)
```

Comment:
Not until it's accepted?

Reply:
Yeah, I mixed up the status of this PR (draft/not ready) with the status of the ADR (still `Proposed` if it was to be merged).